README.md

```diff
@@ -21,13 +21,33 @@ Viewing party is an application in which users can explore movie options and cre
 ## Built With
 - Ruby on Rails
 - JQuery  
+
+## Important Gems:
+[faraday](https://github.com/lostisland/faraday)
+[webmock](https://github.com/bblimke/webmock)
+[vcr](https://github.com/vcr/vcr)
+[figaro](https://github.com/laserlemon/figaro)
+[travis](https://docs.travis-ci.com/user/languages/ruby/)
+
 ## Getting Started
 1. Get an api key from [The Movie Database API](https://developers.themoviedb.org/3/getting-started/authentication)
 1. Fork and clone the repo
 2. Install any missing gems from gem file
 3. Install gem packages with `bundle install`
 4. Setup the database with `rails db:{create,migrate,seed}`
-5. Run RSpec with `bundle exec rspec` (All 97 tests should be passing)
+5. In config/application.ymal paste:
+`MOVIEDB_API_KEY: "<YOUR API KEY>"
+NYT_MOVIE_REVIEWS_API_KEY: "<YOUR API KEY>"`
```

Comment:
I don't think we need this line.

Reply:
My bad, I copied from the wrong application.yml file